src/legacy/core_plugins/vis_type_vega/public/vega_visualization.js

```diff
@@ -72,7 +72,7 @@ export const createVegaVisualization = ({ serviceSettings }) =>
      */
     async render(visData, visParams, status) {
       if (!visData && !this._vegaView) {
-        toastNotifications.addWarning(
+        this.notifications.toasts.addWarning(
```

Comment:
Maybe instead of `this.notifications`, what do you think about using `getNotifications()` here?

Reply:
np =) done